server/src/main/scala/net/wiringbits/repositories/models/Table.scala

```diff
@@ -0,0 +1,7 @@
+package net.wiringbits.repositories.models
```

Comment:
Given that this is core-like functionality, we'd likely prefer a different package, do not worry about it right now, just take a note for the future